src/numpy.jl

```diff
@@ -175,14 +175,20 @@ const npy_typestrs = Dict( "b1"=>Bool,
 #########################################################################
 # no-copy conversion of Julia arrays to NumPy arrays.
 
-function PyObject{T<:NPY_TYPES}(a::StridedArray{T})
+# In some cases, an array is already in row-major order (e.g., RGB
+# images) and it is useful to reverse the order of dimensions passed
+# to PyArray_New.  In these cases, set revdims=true.
+
+function PyObject{T<:NPY_TYPES}(a::StridedArray{T}, revdims::Bool=false)
     try
```

Comment:
Good catch.   For now, it can throw an error if `revdims=true` but in the future `array2py` could take an extra arg.